client/src/App.jsx

```diff
@@ -264,8 +272,8 @@ class App extends Component {
               path="product/:id"
               element={<ProductDetails addToCart={this.addToCart} />}
             />
-            <Route path="/error" element={'Server Error'} />
-            <Route path="*" element={'Page Not Found'} />
+            <Route path="/error" element={<serverError />} />
```

Comment:
Good job here.
Just edit the `serverError` and `notFound` to use PascalCase ==> `ServerError', `NotFound`